source/guide/3.2/reference-vcloud-manager.md

```diff
@@ -68,7 +68,7 @@ The blueprint contains the following nodes:
   - *management_network* - A vCloud routed network for communication between the Cloudify manager and agent machines.
   - *management_network_internet_access* - SNAT rule to enable internet access from management network.
   - *management_port* - connection between Manager Server and Management Network.
-  - *manager_floating_ip* - A floating IP which will be associated with the Cloudify manager machine.
+  - *management_server_internet_access* - A public IP which will be associated with the Cloudify manager machine.
```

Comment:
why was the node's name modified? i find the new name to be less clear than the previous one